yb-voyager/cmd/importData.go

```diff
@@ -1055,19 +1061,34 @@ func getYBSessionInitScript() []string {
 	fileScanner := bufio.NewScanner(varsFile)
 
 	var curLine string
+	var sessionVars []string
 	for fileScanner.Scan() {
 		curLine = strings.TrimSpace(fileScanner.Text())
-		sessionVars = append(sessionVars, curLine)
+		if checkSessionVariableSupport(curLine) {
+			sessionVars = append(sessionVars, curLine)
+		}
 	}
 
-	//Only override the file if the flags are explicitly true (default false)
-	if enableUpsert {
-		sessionVars = append(sessionVars, enableUpsertCmd)
+	return sessionVars
+}
+
+func checkSessionVariableSupport(sqlStmt string) bool {
+	conn, err := pgx.Connect(context.Background(), target.GetConnectionUri())
+	if err != nil {
+		utils.ErrExit("error while creating connection for checking session parameter(%q) support: %v", sqlStmt, err)
 	}
-	if disableTransactionalWrites {
-		sessionVars = append(sessionVars, disableTransactionalWritesCmd)
+	defer conn.Close(context.Background())
+
+	_, err = conn.Exec(context.Background(), sqlStmt)
+	if err != nil {
+		if !strings.Contains(err.Error(), "unrecognized configuration parameter") {
+			utils.ErrExit("error while executing sqlStatement=%q: %v", sqlStmt, err)
+		} else {
+			utils.PrintAndLog("Warning: %q is not supported: %v", sqlStmt, err)
```

Comment:
Just log.
The message is not important for the users.

Reply:
ok